File: web3rpc/rpc-specs/code-samples/python/kaia/configuration/getParams.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

blockNumber = 89

w3 = Web3(Web3.HTTPProvider(host))
kaia_response = w3.kaia.get_params(blockNumber)

print(kaia_response)

File: web3rpc/rpc-specs/code-samples/javascript/kaia/configuration/getParams.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const blockNumber = 89;

  provider.kaia
    .getParams(blockNumber, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
